js/assessment.js

```diff
@@ -247,6 +247,32 @@ define([
             }
         },
 
+        _setupQuestionNumbering: function() {
+            var getRelatedQuestions = function(model) {
+                var currentAssessmentId = model._assessmentId;
+                var currentAssessment =  Adapt.assessment.get(currentAssessmentId);
+                return currentAssessment.getState().questionModels;
+            };
+
+            Handlebars.registerHelper("questionNumber", function getQuestionNumber() {
+                var model = this.view ? this.view.model.attributes : this;
```

Comment:
As before, please use the built-in APIs, i.e. `.attributes` → `.toJSON()`.

Also, no longer a model, so the variable may be better named `data` throughout.